doc/nrf/working_with_nrf/nrf70/developing/regulatory_support.rst

```diff
@@ -0,0 +1,358 @@
+.. _ug_nrf70_developing_regulatory_support:
+
+Operating with regulatory support
+#################################
+
+.. contents::
+   :local:
+   :depth: 2
+
+The nRF70 Series devices operate in free radio spectrum bands (2.4 GHz and 5 GHz).
+The supported channels in each band need to adhere to worldwide regulatory operation rules.
+The regulatory rules vary based on the country.
+The rules are framed on the basis of the following important parameters:
+
+* Channel scan mode (Active/Passive)
+* Dynamic Frequency Selection (DFS) support
+* Maximum transmit power
+
+Regulatory Database
+*******************
+
+The regulatory database used by the nRF70 Series captures regulatory rules in 173 different countries.
+For more information about the individual country rules, see Wi-Fi regulatory channel rules.
```

Comment:
Should `see Wi-Fi regulatory channel rules` be a link to another document where the rules are listed?

Reply:
Yes, this will be updated once the link to the Infocenter doc is available.
@rajb9: Can you provide the link?

Reply:
Location is not yet decided, will provide the link.

Reply:
As discussed with @rajb9, removed line 23. It will be updated once the link to the document is available.